.github/workflows/dashboard-mainnet.yml

```
@@ -0,0 +1,96 @@
+name: Token Dashboard / Mainnet
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - releases/mainnet/**
+  release:
+    types:
+      - "published"
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build for mainnet
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+          cache: "yarn"
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      # FIXME: It's work in progress, the contracts are not yet published.
+      # - name: Resolve latest mainnet contracts
+      #   # For PR we expect the dependencies are already set to correct versions.
+      #   if: github.event_name != 'pull_request'
+      #   run: |
+      #     yarn upgrade \
+      #       @threshold-network/contracts@mainnet
+
+      - name: Build
+        run: yarn build
+
+      - uses: actions/upload-artifact@v2
+        with:
+          name: build
+          path: build
+
+  deploy-preview:
+    name: Deploy mainnet preview
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/download-artifact@v2
+
+      - name: Deploy PR mainnet preview to GCP
+        uses: thesis/gcp-storage-bucket-action@v3.1.0
+        with:
+          # FIXME: Consider setting separate uploader keys for each bucket.
+          service-key: ${{ secrets.KEEP_PRD_UPLOADER_SERVICE_KEY_JSON_BASE64 }}
+          project: ${{ secrets.KEEP_PRD_GOOGLE_PROJECT_ID }}
+          bucket-name: preview.dashboard.threshold.network
+          bucket-path: ${{ github.head_ref }}
+          build-folder: build
+
+      - name: Post mainnet preview URL to PR
+        if: github.event_name == 'pull_request'
+        uses: actions/github-script@v3
+        with:
+          script: |
+            github.issues.createComment({
```

We need to call the `issues.createComment` command using `github.rest.` context (["the Octokit context available via github no longer has REST methods directly"](https://github.com/actions/github-script#breaking-changes-in-v5)). Also, we need to use `v5` of the `github-script` action, which supports the new changes.

```suggestion
        uses: actions/github-script@v5
        with:
          script: |
            github.rest.issues.createComment({
```

FYI, I'm going to apply same change to the https://github.com/keep-network/keep-core/blob/main/.github/workflows/dashboard-mainnet.yml workflow, which also uses the `github-script` action.